roles/chrony/defaults/main.yml

```diff
@@ -12,7 +12,7 @@ chrony_servers:
   - 3.de.pool.ntp.org
 
 # NTP server options.
-chrony_server_options: iburst
+chrony_server_options: iburst nts
```

Comment:
is this generally supported by most pool servers? or does it fallback to not being used if the server doesn't support it?

Reply:
Doesn't look like it. Then we'd better just document it and only add support for custom certificates so that we can use it internally.